app/views/works/errors.json.jbuilder

```diff
@@ -3,4 +3,5 @@
 @form.errors.each do |key, value|
   # Transform the property name in the model to the value used by the javascript:
   json.set! key == :attached_files ? 'file' : key, [value]
+  json.set! key == :embargo_date ? 'embargo-date' : key, [value]
```

Comment:
won't this second thing cause a duplicate set of keys in the response?

Reply:
Can you suggest an alternative?

Reply:
call json.set only one time. adjust the parameters prior to calling it.

Reply:
Moved this into a helper method.